ApollosProject/apollos-embeds
# Fix youtube videos

## 🐛 Issue


This item should have a youtube video on the page: https://liquid-church.webflow.io/racial-reconciliation?id=MediaContentItem-61d1eb6a-961f-4839-9dee-304a9dc993ad

## ✏️ Solution



Remove filtering out youtube videos, this logic came from TV and is not necessary on web.

## 🔬 To Test



1. Verify regular videos still work. https://apollos-web-embeds-git-fix-youtube-videos-apollos-embeds.vercel.app//?id=sermon-on-the-mount-TWVkaWFDb250ZW50SXRlbS04NGRiNjAzNi1mNmUyLTRlNjItYWFjNy1jZjk1YmQzOWIxYmQ%3D
2. Verify youtube videos work. https://apollos-web-embeds-git-fix-youtube-videos-apollos-embeds.vercel.app//?id=MediaContentItem-61d1eb6a-961f-4839-9dee-304a9dc993ad

## 📸 Screenshots


| Before | After |
| --- | --- |
| <img width="1125" alt="image" src="https://github.com/ApollosProject/apollos-embeds/assets/2528817/21a3231f-fd08-4f38-9b82-ded8d59addd2">|<img width="1029" alt="image" src="https://github.com/ApollosProject/apollos-embeds/assets/2528817/11ff8308-8964-402e-84ac-0140b11bf7dc">|
